src/app/admin/requests/page.js

```diff
@@ -2,13 +2,12 @@ import MapView from "@/app/components/MapView";
 import { createClient } from "../../../../utils/supabase/server";
 import AppointmentListView from "@/app/components/AppointmentListView";
 
-const Requests = async() => {
+const Requests = async({searchParams}) => {
```

Comment:
PRAISE: I love searchParams in NextJS! I have limited experience with it, but I know enough to know it makes things a bit smoother! I'm glad you were able to implement it!